src/mmw/js/src/analyze/views.js

```diff
@@ -67,6 +67,7 @@ var AnalyzeWindow = Marionette.LayoutView.extend({
             App: App,
             model: new coreModels.AreaOfInterestModel({
                 shape: this.model.get('area_of_interest'),
+                place: App.map.get('areaOfInterestName'),
```

Comment:
This was easier than passing the `areaOfInterestName` to the [memoized `AnalyzeTaskModel`](https://github.com/WikiWatershed/model-my-watershed/blob/develop/src/mmw/js/src/analyze/controllers.js#L42), since we already have a reference to `App` here.